packages/schematics/angular/application/schema.json

```diff
@@ -19,12 +19,6 @@
       },
       "x-prompt": "What name would you like to use for the application?"
     },
-    "enableIvy": {
```

Comment:
Since this is experimental, I don't think we need to deprecate.

Reply:
Yeah I think that's ok.

Reply:
Don't we still need this option?  The migration won't run until 9 so the option won't be at the root tsconfig yet.